src/csp.operations.js

```diff
@@ -754,7 +754,7 @@ function pipelineInternal(n, to, from, close, taskFn) {
   const jobs = chan(n);
   const results = chan(n);
 
-  times(n, () => {
+  range(n).forEach(() => {
```

Comment:
Again my goal was not to refactor the code, just reduce the bundle size. This could fall into code "refactoring" not necessarily part of this PR.